wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ParallelCommandGroup.java

```diff
@@ -85,17 +87,15 @@ public final void execute() {
 
   @Override
   public final void end(boolean interrupted) {
-    if (interrupted) {
```

Comment:
Behavior change?

Reply:
Slightly. This makes the parallel group call `end` on each composed command that did not finish on its own during execution. For the base ParallelCommandGroup class, this will never occur - the group itself only ends when all internal commands have ended, so the only scenario in which an internal command hasn't finished is if the group were interrupted. Removing the guard clause means that this is slightly less efficient for the base class because it'll now iterate over each composed command when we know it doesn't need to, but makes the behavior correct for subclasses and lets us keep the method declared `final` to avoid overrides

Reply:
Please add a comment explaining this